src/components/UserMounts/UserMountsPanel.js

```diff
@@ -69,6 +69,7 @@ export const UserMountsPanel = (props) => {
             EXPANSION_MAP[props.selectedExpansion]
           }`}
         >
+          <br />
```

Comment:
I actually think using `br` is bad design-wise. This is hard to maintain. I would rather if we wrapped the tabs with a box/container and gave it margin.

Reply:
See my [PR](https://github.com/sofia819/mount_recorder_client/pull/34) for suggested changes.

Reply:
```<br />``` removed, replaced with ```<box m={1}>```